focus-android/app/metrics.yaml

```diff
@@ -56,11 +60,12 @@ browser:
     data_reviews:
       - https://github.com/mozilla-mobile/focus-android/pull/5065#issuecomment-894328647
       - https://github.com/mozilla-mobile/focus-android/pull/7418#issuecomment-1195518264
+      - https://github.com/mozilla-mobile/firefox-android/pull/3320
     data_sensitivity:
       - technical
     notification_emails:
       - android-probes@mozilla.com
-    expires: 119
+    expires: 123
```

Comment:
I don't see this probe in the Request for Data Collection Renewal - should it be?